style.css

```diff
@@ -91,16 +109,450 @@
+
+  .ruby {
+    position: absolute;
+    display: flex;
+    width: 100px;
+    height: 20px;
+    left: 99px;
+    top: 60px;
+    background: #98dede;
+  }
+
+  li.ruby {
+    font-size: 12px;
+    color: #333;
+    font-weight: bold;
+    font-family: Arial, sans-serif;
+    padding: 3px 0 0 3px;
+  }
+
+  .javas {
+    position: absolute;
+    display: flex;
+    width: 74px;
+    height: 20px;
+    left: 210px;
+    top: 60px;
+    background: #98dede;
+  }
+
+  li.javas {
+    font-size: 12px;
+    color: #333;
+    font-weight: bold;
+    font-family: Arial, sans-serif;
+    padding: 3px 0 0 3px;
+  }
+
+  .stp {
+    position: absolute;
+    display: flex;
+    width: 178px;
+    height: 44px;
+    left: 20px;
+    top: 95px;
+    border: 1px solid #fff;
+  }
+
+  .buttn {
+    position: absolute;
+    width: 178px;
+    height: 44px;
+    left: 20px;
+    top: 95px;
+    background-color: transparent;
+    border: 1px solid white;
+    font-size: 18px;
+    color: white;
+    padding: 0 30px 0 0;
+    background-image: url(/Asset/union.svg);
+    background-size: 25px;
+    background-repeat: no-repeat;
+    background-position: center right;
+  }
+
+  .about {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
+  }
+
+  .about h1 {
+    position: absolute;
+    display: flex;
+    width: 30%;
+    height: 50px;
+    left: 16px;
+    top: 2960px;
+    font-family: "Playfair Display", monospace;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 50px;
+    line-height: 100%;
+
+    /* identical to box height, or 50px */
+
+    color: #1a2236;
+  }
+
+  .about p {
+    position: absolute;
+    display: flex;
+    width: 75%;
+    height: 126px;
+    left: 20px;
+    top: 3025px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 21px;
+    color: #1a2236;
+  }
+
+  .about button {
+    position: absolute; /* set the button to relative position */
+    width: 85%;
+    height: 44px;
+    left: 20px;
+    top: 3180px;
+    background-color: #1a2236;
+    border: 1px solid white;
+    font-size: 18px;
+    color: white;
+    display: flex;
+    align-items: center;
+    padding: 0 30px 0 95px;
+  }
+
+  .about button::before {
+    content: ""; /* add content to the pseudo-element */
+    background-image: url(/Asset/download.svg);
+    background-size: 25px;
+    background-repeat: no-repeat;
+    background-position: left center;
+    width: 25px; /* set the width and height of the pseudo-element */
+    height: 25px;
+    position: absolute; /* position the pseudo-element absolutely */
+    left: 220px; /* set the left offset to position the image */
+  }
+
+  .card1 {
+    position: absolute;
+    display: flex;
+    width: 85%;
+    height: 289px;
+    left: 16px;
+    top: 3280px;
+    background: #1c528a;
+    background-image: url(/Asset/Rectangle.png);
+    background-image: url(/Asset/hash.png);
```

Comment:
- [ ] As requested by the last code reviewer, please 🙏🏽 add the images in the corresponding div of HTML code because they are part of the section not decorative as on the home page. It will be easier to adjust the position to be similar to the Figma template. 👇🏽 
- [ ] Kindly review the path of the images because those are not able to see in the browser for a better user experience.

> Looks like the space is so large between each project image and the project item detail content, please stick to the Figma design as much as possible, it should be no space between them.

![image](https://user-images.githubusercontent.com/58642949/220754509-6ffaf959-54a0-4c90-8572-fc5b5de08acf.png)

Reply:
@franclobo 
1.The image part was marked by the reviewer as optional, I am afraid you don't have the write to change it to mandatory.
2.For your review two the image is loading okay you suppose to wait it. since it was okay by both reviewer and also i can attach you the image if you want.